### PR TITLE
Support icraf consent ad 

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/ws_correct_dataset.json [hunk 1]
```diff
@@ -411,7 +411,7 @@
       ]
     },
     {
-      "CodeID": "code-803a759e",
+      "CodeID": "code-cb5a298b",
```
Comment (as2388) [inline code_schemes/ws_correct_dataset.json]: Why is this changing?

File: configuration/kiambu_pipeline_config.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
         "gpsdd_kiambu_s01e05_activation",
         "gpsdd_kiambu_s01e06_activation",
         "gpsdd_kiambu_s01e07_activation",
-        "gpsdd_kiambu_s01e08_activation"
+        "gpsdd_kiambu_s01e08_activation",
+        "gpsdd_kiambu_s01e05_Icraf_Kenyanpool_sms_ad"
```
Comment (as2388) [inline configuration/kiambu_pipeline_config.json:19]:
```suggestion
        "gpsdd_kiambu_s01e05_icraf_kenya_pool_sms_ad"
```
unless this rename is going to cause problems in TextIt

**Review by as2388 — APPROVED** [review]

File: code_schemes/ws_correct_dataset.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/ws_correct_dataset.json]: Thanks for catching this I updated on the wrong CodeID. Fixed thanks.

File: configuration/kiambu_pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/kiambu_pipeline_config.json:19]: yup, not quite sure how textit will handle this. will keep as is

**Pull request merged**